Microsoft.Toolkit.Uwp.UI.Controls.Layout/GridSplitter/GridSplitter.Data.cs

```diff
@@ -57,87 +57,6 @@ public enum GridResizeBehavior
             PreviousAndNext
         }
 
-        /// <summary>
-        ///  Enum to indicate the supported gripper cursor types.
-        /// </summary>
-        public enum GripperCursorType
```

Comment:
BREAKING CHANGE: this enum is pretty similar to the `Windows.UI.Core.CoreCursorType` enum, to where the value gained by having a GridSplitter-specific enum (to do the kind of resizing cursor behavior we want to do) isn't warranted, so why not just use `CoreCursorType` in the first place?  

- This GridSplitter-specific enum adds an additional element, called `Default`, with value -1.

But, still a breaking change possibility if we do make the switch to the core enum, as evidenced in one example with the `FrameworkElementExtensionsCode.bind` sample page, which references the GridSplitter-specific enum `Default` value.

Reply:
Yeah, fine if we get rid of this enum for the cursor as we offloaded that to our other extension.